exercises/practice/bob/.approaches/if-statements/content.md

```diff
@@ -1,88 +1,87 @@
 # `if` statements
 
 ```java
-import java.util.function.Predicate;
```

Comment:
Sorry, I think there might been a bit of misundstanding, as I didn't mean to copy Python's approaches entirely. I was hoping keep the original `if-statement` approach as it was before your changes and your approach added as `if-statement-using-methods` (or something similar). Sorry for not being clearer earlier.